### PR TITLE
Adding the workflow to create releases with changelog

### PR DESCRIPTION
**Pull request by Lyaaaaaaaaaaaaaaa**
- This workflow happens when a tag is pushed.
- Changelog will list all commits and commits' summary.
- Release's body will be the previously generated changelog.

**You might want to add `on pull request` so it will create a release on pull requests. I'm not sure of how tags works on pulls request**
Moreover, you might want to create more branches and keep the master for production only (it depends of how you organise the repository though.)

You can preview the result of my [tests here](https://github.com/Lyaaaaaaaaaaaaaaa/godot-console/releases/tag/v0.0.2)

Closes: #42

### CODE REVIEW

**Comment by quentincaffeino** [comment]
Hello and thank you very much for your help.

I'll check it out more in-depth and merge a bit later. But from what I see now from your tests it looks perfect!

**Comment by quentincaffeino** [comment]
So I thought that best way to test it just to merge, and I did. Also I've had several tags locally for older versions which I pushed, and I see nothing. Does this work only for tags for commits that were added after action workflow?

**Comment by Lyaaaaaaaaaaaaaaa** [comment]
I fear it only works for tags after it. I pushed 2 tags at the same time on my test repository and it created two releases but these releases were after the commit which added the workflow.
Maybe you can try making a branch to test it. Make a test tag in this branch after updating the workflow file:
- **Line 34** set `draft = true` so it won't publish an actual release but just a draft for us to test it.

File: .github/workflows/automatic_release.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        id: create_release
+        uses: actions/create-release@latest # Uses https://github.com/marketplace/actions/create-a-release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          body: |
+            ${{ steps.Changelog.outputs.changelog }}
+          draft: false
```
Comment (Lyaaaaaaaaaaaaaaa) [inline .github/workflows/automatic_release.yml:34]: Set `draft = true` to not publish a release by mistake

**Comment by quentincaffeino** [comment]
Yeah thanks, that worked)

**Comment by quentincaffeino** [comment]
@all-contributors please add @Lyaaaaaaaaaaaaaaa for infra

**Pull request merged**